apis/datadoghq/common/envvar.go

```diff
@@ -98,6 +98,8 @@ const (
 	DDKubeResourcesNamespace                             = "DD_KUBE_RESOURCES_NAMESPACE"
 	DDKubeStateMetricsCoreConfigMap                      = "DD_KUBE_STATE_METRICS_CORE_CONFIGMAP_NAME"
 	DDKubeStateMetricsCoreEnabled                        = "DD_KUBE_STATE_METRICS_CORE_ENABLED"
+	DDLanguageDetectionEnabled                           = "DD_LANGUAGE_DETECTION_ENABLED"
+	DDLanguageDetectionReportingEnabled                  = "DD_LANGUAGE_DETECTION_REPORTING_ENABLED"
```

Comment:
I don't believe `DDLanguageDetectionReportingEnabled` is used anywhere. Do we need this env var